tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py

```diff
@@ -649,6 +649,10 @@ def get_minimum_resources(self, is_gpu_available=False):
             minimum_resources["num_gpus"] = 0.5
         return minimum_resources
 
+    @classmethod
+    def supported_problem_types(cls) -> list[str] | None:
+        return ["binary", "multiclass", "regression", "quantile"]
```

Comment:
Should we use constants instead? (also applies to the implementation of `supported_problem_types` for other models)

Reply:
I think it is a good idea, and that might be something I add such as `ProblemType.binary` so that it is easier to type check rather than `BINARY`.